sdcm/nemesis.py

```diff
@@ -1525,19 +1525,21 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
+        if not (self.cluster.params.get('use_mgmt') or
+                self.cluster.params.get('use_cloud_manager')):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
         cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
-        if cluster_backend == 'aws':
+        if self.cluster.params.get('use_mgmt'):
+            if not self.cluster.params.get('backup_bucket_location'):
+                raise UnsupportedNemesis('backup bucket location configuration is not defined!')
+            backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        else:
+            backup_bucket_location = self.cluster.get_backup_buckets_name()
+        if 'aws' in cluster_backend:
             bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'gce':
+        elif 'gce' in cluster_backend:
```

Comment:
why are you replacing this logic for gce, it might start doing backups for k8s, and they might not be ready for that.

Reply:
I figured that it's probably for the best to change the logic for gce if I do it for aws,but I'll revert it if it could interfere with another run

Reply:
the AWS logic, we should change, but for GCE lets not change anything (at least not yet)